src/graphics/Screen.cpp

```diff
@@ -968,7 +990,11 @@ static void drawTextMessageFrame(OLEDDisplay *display, OLEDDisplayUiState *state
     display->setFont(FONT_SMALL);
     if (config.display.displaymode == meshtastic_Config_DisplayConfig_DisplayMode_INVERTED) {
         display->fillRect(0 + x, 0 + y, x + display->getWidth(), y + FONT_HEIGHT_SMALL);
-        display->setColor(BLACK);
+#if defined(M5STACK_COREBASIC)   || defined(M5STACK_CORE2) 
+    display->setColor(OLEDDISPLAY_COLOR::BLACK);
+#else
+    display->setColor(BLACK);
+#endif
     }
```

Comment:
This seems overly complicated. Why not alias these constants at the top of the file?

Reply:
ok,Thank you for your advice